packages/sitebuilder/src/wizards/first-time-configuration/data/first-time-configuration-screen-data.tsx

```diff
@@ -56,14 +61,55 @@ const industryVerticals: Record<string, string[]> = {
 	Other: []
 };
 
-const personalityOptions: string[] = [
-	'Quirky',
-	'Calm',
-	'Spicy',
-	'Classic',
-	'Elegant',
-	'Funny',
-	'Witty'
+const personalityOptions: TonePersonalityInterface[] = [
+	{
+		name: 'Neutral',
+		description: 'Provides balanced and unbiased information without any specific emotional tone.',
+	},
+	{
+		name: 'Inspirational',
+		description: 'Motivates and inspires the audience, encouraging them to pursue their goals and aspirations.',
+	},
+	{
+		name: 'Funny',
+		description: 'Adds humor and entertainment to the content, aiming to create a lighthearted and amusing experience for the audience.',
+	},
+	{
+		name: 'Upbeat',
+		description: 'Infuses content with positivity, energy, and enthusiasm, leaving a lasting impression on visitors.',
+	},
+	{
+		name: 'Persuasive',
+		description: 'Influences and convinces the audience to take action through compelling and convincing language.',
+	},
+	{
+		name: 'Persuasive',
+		description: 'Influences and convinces the audience to take action through compelling and convincing language.',
+	},
+	{
+		name: 'Conversational',
+		description: 'Mimics natural conversation, making the content relatable and easy to understand.',
+	},
+	{
+		name: 'Trustworthy',
+		description: 'Establishes credibility, reliability, and trust, essential for building strong relationships with customers.',
+	},
+	{
+		name: 'Engaging',
+		description: 'Captivates and holds the attention of the audience through compelling and interactive content.',
+	},
+	{
+		name: 'Informative',
+		description: 'Focuses on providing clear and helpful information about products, services, or topics of interest.',
+	},
+	{
+		name: 'Friendly',
+		description: 'Creates a warm and approachable atmosphere, fostering a personal connection with the audience.',
+	},
+	{
+		name: 'Professional',
+		description: 'Conveys expertise, reliability, and competence. Suitable for businesses and professional services.',
+	},
 ];
```

Comment:
We'll need a `value` property for each tone to match the values expected to be consumed by the Prophecy endpoint. The Prophecy endpoint will return an error if the tone value doesn't match exactly on of these values:
```
CONVERSATIONAL
ENGAGING
FRIENDLY
FUNNY
INFORMATIVE
INSPIRATIONAL
NEUTRAL
PERSUASIVE
PROFESSIONAL
TRUSTWORTHY
UPBEAT
```